website/app/layout.tsx

```diff
@@ -34,6 +34,17 @@ const RootLayout: FC<{
         docsRepositoryBase:
           'https://github.com/the-guild-org/the-guild-components/tree/main/website',
       }}
+      navbarProps={{
+        navLinks: [
+          {
+            href: '/docs',
+            children: 'Documentation',
+          },
+        ],
+        searchProps: {
+          placeholder: 'Search...',
+        },
```

Comment:
scope creep, but I just realized it would be neat for it to say "Search documentation" on large screens and "Search" on smaller ones to fill the space, like with `before` pseudoelement

Reply:
I think too tricky because the placeholder should be always `string`